packages/graphql/src/auth/get-jwt.ts

```diff
@@ -68,8 +70,33 @@ function getJWT(context: Context): any {
             debug("Skipping verifying JWT as noVerify is not set");
 
             result = jsonwebtoken.decode(token);
-        } else {
-            debug("Verifying JWT");
+        } else if (jwtConfig.jwkEndpoint) {
+            debug("Verifying JWT using OpenID Public Key Endpoint");
+
+            // Create a JWK Client with a rate limit that
+            // limits the number of calls to our JWK endpoint
+            client = new JwksClient({
+                jwksUri: jwtConfig.jwkEndpoint || "",
```

Comment:
Do we really need a default value here? At this point the `jwtConfig.jwkEndpoint` should always be set, right?